src/app/routes.tsx

```diff
@@ -156,6 +158,18 @@ const flightRecorderRoutes: IAppRoute[] = [
   },
 ];
 
+const diagnosticsRoutes: IAppRoute[] = [
+  {
+    component: Diagnostics,
+    label: 'Thread Dumps',
+    path: toPath('/diagnostics'),
+    title: 'Thread Dumps',
+    description: 'Create and view thread dumps on single target JVMs.',
+    navGroup: DIAGNOSTICS,
+    navSubgroup: CAPTURE,
```

Comment:
I think this should actually be under the `ANALYZE` subgroup, since the Thread Dump table located here is analogous to the (JFR) Archives tables. Diagnostics > Capture would be a view that allows the user to request a new thread dump to be collected, so it would probably just look a lot like a scaled-up version of the dashboard card you've implemented.